src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Factory/EntityTypeFactory.php

```diff
@@ -65,27 +65,41 @@ public function createCreateForm(string $type, Service $service, string $environ
             case ($type === Constants::TYPE_OPENID_CONNECT_TNG_RESOURCE_SERVER):
                 $command = SaveOidcngResourceServerEntityCommand::forCreateAction($service);
                 $command->setEnvironment($environment);
-                return $this->formFactory->create(OidcngResourceServerEntityType::class, $command, $this->buildOptions($environment));
+                return $this->formFactory->create(
+                    OidcngResourceServerEntityType::class,
+                    $command,
+                    $this->buildOptions($environment)
+                );
         }
 
         throw new InvalidArgumentException("invalid form type requested: " . $type);
     }
 
-    public function createEditForm(ManageEntity $entity, Service $service, string $environment)
+    public function createEditForm(ManageEntity $entity, Service $service, string $environment, $isCopy = false)
```

Comment:
I'm uncertain it is valuable to add the isCopy parameter to this method. While editing the entity, you are never in copy context right?

Reply:
you are: when copying an entity it creates an edit form with the values of the entity being copied prefilled.  There's a check in the entity edit form to see whether or not it's a copy or an edit, but that did not work anymore (not sure if that's recent or not, i think it's a side-effect of switching to manageIds for entity).